api/proto/common.go

```diff
@@ -97,3 +98,10 @@ func ConstructDRandMessage(payload []byte) []byte {
 	byteBuffer.Write(payload)
 	return byteBuffer.Bytes()
 }
+
+// ConstructStakingMessage creates a message with the payload and returns as byte array.
+func ConstructStakingMessage(payload []byte) []byte {
```

Comment:
the raw message buffer has to be wrapped in constructp2pmessage function call to add the 5 bytes header. otherwise, the receiver won't be able to interpret the message.  please fix it.

Reply:
Forgot it. Thanks. Fixed